docs/framework/release-notes/2024/04-april-security-and-quality-rollup.md

```diff
@@ -0,0 +1,103 @@
+# April 2024 Security and Quality Rollup
+
+_Released April 9, 2024_
+
+## Summary of What's New in this Release
+
+### Security improvements
+
+#### CVE-2024-21409 – Remote Code Execution Vulnerability
+
+This security update addresses a remote code execution vulnerability detailed in [CVE 2024-21409](https://portal.msrc.microsoft.com/en-US/security-guidance/advisory/CVE 2024-21409).
+
+#### Defense in Depth Vulnerability
+
+This security update addresses an issue where version of the OSS zlib library is out of date.
+
+#### Defense in Depth Vulnerability
+
+This security update addresses an issue in AIA fetching process.
+
+### Quality and reliability improvements
+
+This release contains the following quality and reliability improvements.
+
+#### ASP.NET
++ Addresses an issue with JavaScriptSerializer where after installing the January Security and Quality update there is a performance degradation. (*applies to: .NET Framework 4.6.2, 4.7, 4.7.1,4.7.2, 4.8, 4.8.1*)
+
+#### CLR
++ Addressed an issue with thread pool recycled lists become unresponsive on multi-CPU-group computers (*applies to: .NET Framework 4.6.2, 4.7, 4.7.1, 4.7.2, 4.8, 4.8.1*)
```

Comment:
```suggestion

Addressed an issue with thread pool recycled lists becoming unresponsive on multi-CPU-group computers (*applies to: .NET Framework 4.6.2, 4.7, 4.7.1, 4.7.2, 4.8, 4.8.1*).
```

Reply:
Why not simply say .NET Framework 4.6.2 and newer versions?